modules/service/src/main/scala/lucuma/odb/service/ObservationService.scala

```diff
@@ -89,13 +87,9 @@ sealed trait ObservationService[F[_]] {
   ): F[Result[Observation.Id]]
 
   def createObservation(
-    input: CreateObservationInput
+    input: AccessControl.CheckedWithId[ObservationPropertiesInput.Create, Program.Id]
   )(using Transaction[F]): F[Result[Observation.Id]]
 
-  def selectObservations(
-    which: AppliedFragment
-  )(using Transaction[F]): F[List[Observation.Id]]
-
```

Comment:
This service method was only used once and is inlined in `ObservationWorkflowService` below.